File: src/main/java/com/lee/guess/GameFor.java
package com.lee.guess;

import java.util.Random;
import java.util.Scanner;

public class GameFor {

    public static void main(String[] args) {

        int secret = new Random().nextInt(10)+1;
        Scanner scanner = new Scanner(System.in);
        System.out.println(secret);
        for (int i = 1 ; i <= 4 ; i++){
            System.out.println("Please input your number("+i+"/4) : ");
            int number = scanner.nextInt();
            System.out.println("猜第"+i+"次");
            if (number > secret){
                System.out.println("lower");
            }else if(number < secret){
                System.out.println("higher");
            }else{
                System.out.println("Great,number is "+number);
                break;
            }

            if (number == -1)
            {
                break;
            }


        }

    }
}

File: src/main/kotlin/com/lee/GameFor.kt
package com.lee

import java.util.*

fun main() {

    var sercet = Random().nextInt(10)+1
    println("$sercet")
    val scanner = Scanner(System.`in`)
    var isMatch =false;
    for(i in 1..4){
        print("Please input your number($i/4):")
        var number = scanner.nextInt()
        if (number>sercet){
            println("lower")
        }else if (number<sercet){
            println("higher")
        }else{
            println("Great number is $number")
            isMatch = true
            break
        }

    }
    if (!isMatch) println("not guess number , number is $sercet")



}